File: Python/Daily Coding Problem#21.py
def findMaximumRooms(time):
    prefix = [0] * 100001
    for i in range(0, len(time)):
        prefix[time[i][0]] += 1
        prefix[time[i][1] + 1] -= 1
    ans = prefix[0]
    for i in range(1, 100001):
        prefix[i] += prefix[i-1]
        ans = max(prefix[i], ans)
    return ans


size = int(input("Enter size of an array"))
time = []
print("Enter array elements rowwise:")
for i in range(0, size):
    a = []
    for i in range(0, 2):
        a.append(int(input()))
    time.append(a)

print(findMaximumRooms(time))

File: Javascript/Daily Coding Problem#21.js
const findMaximumRooms = (time) => {
    let prefix = new Uint8Array(100001);
    for (let i = 0; i < time.length; i++) {
        prefix[time[i][0]]++;
        prefix[time[i][1] + 1]--;
    }
    let ans = prefix[0];
    for (let i = 1; i < 100001; i++) {
        prefix[i] += prefix[i - 1];
        ans = Math.max(ans, prefix[i]);
    }
    return ans;
}

let arr = [[0, 50], [30, 75], [60, 150]];
console.log(findMaximumRooms(arr));